cpp/oneapi/dal/detail/array_impl.hpp

```diff
@@ -245,7 +269,7 @@ class array_impl : public base {
 #ifdef ONEDAL_DATA_PARALLEL
     std::optional<data_parallel_policy> dp_policy_;
 #endif
-};
+}; // namespace v1
```

Comment:
```suggestion
};
```
It's a brace of `array_impl`